bindings/node/lib/stateMachine.js

```diff
@@ -432,15 +432,16 @@ module.exports = function (modules) {
      */
     markCommand(client, ns, command, callback) {
       const bson = this.bson;
+      const options = { promoteLongs: false };
```

Comment:
Just for context, I put in the `promoteValues: false` here to be explicit about the fact that we always want to be sure to forward the *exact* message from mongocryptd to libmongocrypt.

Reply:
I brought it back now.